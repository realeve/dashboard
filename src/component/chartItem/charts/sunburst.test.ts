import init from './lib.test';
init(__filename);
