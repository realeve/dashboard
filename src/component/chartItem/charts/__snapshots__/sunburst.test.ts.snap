// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`基础信息测试-API配置项 1`] = `
Object {
  "legend": 0,
  "y": 2,
}
`;

exports[`基础信息测试-默认渲染设置/边距 1`] = `
Object {
  "renderer": "svg",
}
`;

exports[`基础信息测试-默认配置项 1`] = `
Object {
  "border": true,
  "borderColor": "#080226",
  "borderRadiusInner": 0,
  "borderRadiusOutter": 0,
  "borderWidth": 1,
  "fontSize": 12,
  "innerRadius": 0,
  "needRerverse": false,
  "theme": "cbpc",
}
`;
